src/components/table/use-cases/generations.tsx

```diff
@@ -354,8 +443,12 @@ export default function GenerationsTable({ projectId }: GenerationsTableProps) {
           traceName: generation.traceName,
           startTime: generation.startTime.toLocaleString(),
           endTime: generation.endTime?.toLocaleString() ?? undefined,
+          timeToFirstToken:
+            generation.completionStartTime?.toLocaleString() ?? undefined,
           latency: generation.latency === null ? undefined : generation.latency,
-          cost: generation.calculatedTotalCost,
+          totalCost: generation.calculatedTotalCost,
+          inputCost: generation.calculatedInputCost,
+          outputCost: generation.calculatedOutputCost,
```

Comment:
Am i missing something or are these missing on `GenerationsTableRow`?

Reply:
You are right. I added them. 🙂